### DISCUSSION TITLE
feature consideration: bring element into view

### DISCUSSION THREAD

**Discussion by justin-hackin** [chrvadala/react-svg-pan-zoom#193] [Q&A] [answered]
### Features:

It would be great if I could zoom in and center on a particular svg element by providing its bounding box

**Comment by chrvadala** [comment, marked as answer]
Did you look at library methods https://chrvadala.github.io/react-svg-pan-zoom/?path=/story/react-svg-pan-zoom--call-methods
It seems that what you're asking is already there

**Comment by chrvadala** [comment]
Issue moved on discussions section

**Reply by SmileHub** [reply]
Look through samples:
```
..
Viewer.current.fitToViewer();
...
<UncontrolledReactSVGPanZoom
					ref={Viewer}
...
</UncontrolledReactSVGPanZoom>

```
